src/style.css

```diff
@@ -1,3 +1,88 @@
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--background-color);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu_box {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.menu_item {
+  font-family: Roboto, sans-serif;
```

Comment:
You're specifying the `font-family` here which is already defined for the `body`. If the intention is to have the same font throughout the website, you don't need to redeclare it for `.menu_item`. Consider removing it to reduce redundancy and maintain consistency.